frontend/src/components/Logo/Logo.tsx

```diff
@@ -1,7 +1,31 @@
+import { Body, Card, Dropdown, H3, Hr } from "@maykin-ui/admin-ui";
+
 export type LogoProps = {
   width?: number;
 };
 
-export function Logo({ width = 128 }: LogoProps) {
+export function Logo({ width }: LogoProps) {
   return <img src="/logo.svg" alt="Open Archiefbeheer Logo" width={width} />;
 }
+
+export function LogoDropdown({ width = 128 }: LogoProps) {
```

Comment:
I think we should show a modal, maybe use `useDialog` for this, which is undocumented in admin ui for now, so maybe add a story to document it as well.

Reply:
IMO the profile button acts as a menu, where the modal is somewhat like a traditional about box: 
![](https://i.sstatic.net/X9Rr8.png)

So having this "inconsistently" is OK with me.